src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntryHelper.js

```diff
@@ -331,6 +338,43 @@
         return totalsMatchIfRequired;
     },
 
+    /**
+     * @description: Navigates to the record view of the Data Import record
+     * @param row: Information about which row the action was called from
+     */
+    handleViewRowAction: function(component, row) {
+        let navEvt = $A.get("e.force:navigateToSObject");
+        navEvt.setParams({
+            recordId: row.Id,
+            slideDevName: 'detail'
+        });
+        navEvt.fire();
+    },
+
+    /**
+     * @description: Deletes the data import record displayed in a given row
+     * @param row: Information about which row the action was called from
+     */
+    handleDeleteRowAction: function(component, row) {
+        let self = this;
```

Comment:
Just curious, what's wrong with just using 'this'?

Reply:
The definition of `this` changes inside of nested functions (like the callback here), though I see that it's passed in as a parameter. I added it because `this` didn't seem to be working. :)